src/binfhe/include/binfhecontext.h

```diff
@@ -74,7 +74,8 @@ class BinFHEContext : public Serializable {
    * @return creates the cryptocontext
    */
     void GenerateBinFHEContext(uint32_t n, uint32_t N, const NativeInteger& q, const NativeInteger& Q, double std,
```

Comment:
Please move the numAutoKeys parameter to the end. Otherwise, it will break the code using the existing API.